### ISSUE TITLE
Windows Install

### ISSUE THREAD

**Issue by ChrisPal323** [foolwood/SiamMask#188]
I have been trying to get the demo working on my windows machine, and have not been able to figure out the commands to get a successful install. I get all the way to executing the code and right when I get to executing the demo I get this error:

File "../../tools/demo.py", line 7, in <module> from tools.test import * ModuleNotFoundError: No module named 'tools'

Any clue how to fix this on a windows machine? I think it has to do with the export commands I have been doing wrong on windows. I have been using the set command and still nothing. Any clue why?

Thanks for the help!

**Comment by asuleyman** [comment]
I have the same problem! 
did you managed to figure it out ?

**Comment by ChrisPal323** [comment]
Yes, you should change the path in environmental settings on windows. The
same as in the GitHub instructions, but in the windows envriomnetal
settings.

**Comment by asuleyman** [comment]
Thank you. It took me a while to set up the environment.

**Comment by Soyad-yao** [comment]
i got the same error.
I have added environment variables but keep getting this error.
Can you illsutrate how to solve it?
Thanks!
